aggregator/apps/api/v1/api_client.py

```diff
@@ -186,6 +186,7 @@ def filter_single_election(self, election):
             election["organisation"].pop("url", None)
 
         if "division" in election and election["division"]:
+            election["division"].pop("division_subtype", None)
```

Comment:
As I was documenting each field in the schema, I realised that the content of this field doesn't really match the name and isn't terribly useful, so I've decided to remove it. We can always review/put it back in if we make it better in future.